docs/imagecustomizer/concepts/sysext.md

```diff
@@ -0,0 +1,268 @@
+---
+parent: Concepts
+nav_order: 6
+---
+
+# System extension
+
+System extensions (sysext) are a systemd feature that allows extending an operating
+system without modifying the base system. They are useful for immutable or read-only
+OS environments, enabling modular functionality while preserving system integrity.
+
+Key Characteristics:
+- Dynamic Overlay: Files from a sysext image are dynamically overlaid onto /usr and /opt
+  at runtime.
+- Immutable Base System: The base OS remains unchanged, allowing independent updates.
+- Modular & Flexible: Extensions can be added, removed, or updated without modifying
+  core OS files.
+- Verification & Security: Sysext images can include dm-verity and cryptographic
+  signatures to ensure integrity and authenticity.
+- No Persistent Changes: Once deactivated, system extensions disappear from the
+  filesystem without leaving traces.
+
+# System Extension Image
```

Comment:
Only a single heading should be a title (i,e. `#`). All the rest should be sub-headings.